packages/domains-table/src/domains-table/domains-table-ssl-cell.tsx

```diff
@@ -4,59 +4,67 @@ import { lock } from '@wordpress/icons';
 import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
 
-interface DomainsTableSSLCellProps {
+interface DomainsTableSslCellProps {
 	domainManagementLink: string;
 	sslStatus: DomainData[ 'ssl_status' ];
+	hasWpcomManagedSslCert?: DomainData[ 'has_wpcom_managed_ssl_cert' ];
 }
 
-export default function DomainsTableSSLCell( {
+export default function DomainsTableSslCell( {
 	domainManagementLink,
 	sslStatus,
-}: DomainsTableSSLCellProps ) {
+	hasWpcomManagedSslCert,
+}: DomainsTableSslCellProps ) {
 	const translate = useTranslate();
+	// WordPress.com managed subdomains (e.g. *.wordpress.com, *.wpcomstaging.com, etc.)
+	// are covered by a wildcard SSL cert, so we display them as 'Active'.
+	const isActiveSsl = sslStatus === 'active' || hasWpcomManagedSslCert;
+	const isPendingSsl = sslStatus === 'pending' || sslStatus === 'newly_registered';
+	const domainHasSsl = sslStatus !== null || hasWpcomManagedSslCert;
 
-	const isPendingSSL = sslStatus === 'pending' || sslStatus === 'newly_registered';
-
-	const getSSLStatusText = () => {
-		if ( sslStatus === 'active' ) {
+	const getSslStatusText = () => {
+		if ( isActiveSsl ) {
 			return translate( 'Active' );
 		}
-		if ( isPendingSSL ) {
+		if ( isPendingSsl ) {
 			return translate( 'Pending' );
 		}
 		if ( sslStatus === 'disabled' ) {
 			return translate( 'Disabled' );
 		}
 	};
 
+	const getSslStatusClassnames = ( element: 'button' | 'icon' ) =>
+		clsx( `domains-table-row__ssl-status-${ element }`, {
+			[ `domains-table-row__ssl-status-${ element }__active` ]: isActiveSsl,
+			[ `domains-table-row__ssl-status-${ element }__pending` ]: isPendingSsl,
+			[ `domains-table-row__ssl-status-${ element }__disabled` ]: sslStatus === 'disabled',
+		} );
+
+	let button: React.ReactElement | string;
+
+	if ( sslStatus ) {
+		button = (
+			<a
+				className={ getSslStatusClassnames( 'button' ) }
+				href={ `${ domainManagementLink }?ssl-open=true` }
+				onClick={ ( event ) => event.stopPropagation() }
+			>
+				{ getSslStatusText() }
+			</a>
+		);
+	} else if ( hasWpcomManagedSslCert ) {
+		button = <span className={ getSslStatusClassnames( 'button' ) }>{ getSslStatusText() }</span>;
```

Comment:
```suggestion
		button = <span className={ getSslStatusClassnames( 'button' ) }>{ translate( 'Active' ) }</span>;
```
Nit-ish, but since we will only ever render `Active` here – I think we might as well be explicit about that.

Reply:
Good point!